visualc/VS2010/mbedTLS.vcxproj

```diff
@@ -257,14 +257,12 @@
     <ClCompile Include="..\..\crypto\library\blowfish.c" />
     <ClCompile Include="..\..\crypto\library\camellia.c" />
     <ClCompile Include="..\..\crypto\library\ccm.c" />
-    <ClCompile Include="..\..\crypto\library\certs.c" />

```

Comment:
In commit message for "Update VS2010 project according to new shadowing rules", explain what the new shadowing rules are. Or, better yet, consider squashing this commit with "Generating visualc files - let Mbed TLS take precedence over crypto", since it makes sense to change the generation rules and re-generate in one atomic commit (since there would no intermediate commits that might fail tests).